upgrades/3.x/3.12.0/README.adoc

```diff
@@ -0,0 +1,27 @@
+= Upgrade to 3.12.0
+
+== Model
+
+=== Api keys
+
+Before this version, ApiKey contained a *key* attribute, which is the value of the key, and at the same time his database id.
+
+From this version, an ApiKey contains distinct attributes :
+
+- *key* : api key value
+- *id* : api key database unique id
+
+Those 3 Portal API's endpoints now expose distinct *id* and *key* in HTTP response (previously, the *key* attribute was exposed as '*id*') :
+
+- GET /subscriptions/{subscriptionId}
+- POST /subscriptions/{subscriptionId}
+- POST /subscriptions/{subscriptionId}/_renew
+
+== Repository
+
+=== Mongodb
```

Comment:
Can we please change to: 

Starting in version 3.12.0, APIKeys now contain distinct attributes: 

The 3 Portal API endpoints listed below now expose distinct *ID* and *key* in the HTTP response (previously, the *key* attribute was exposed as '*ID*'):

=== MongoDB

Reply:
:+1: done